src/params.rs

```diff
@@ -207,3 +212,39 @@ impl FromStr for GuestMemorySize {
 		Ok(memory_size)
 	}
 }
+
+/// Configure the kernels environment variables.
+#[derive(Debug, Clone)]
+pub enum EnvVars {
+	/// Pass all env vars of the host to the kernel.
+	Host,
+	/// Pass a certain set of env vars to the kernel.
+	Set(HashMap<String, String>),
```

Comment:
I'm not super happy with passing strings here, but using `<&str, &str>` leads to a complete lifetimecalypse, as the lifetime parameters then also need to be explicitly specified for `KernelInfo` -> `UhyveVm` -> `VirtCpu` -> `KvmCpu` -> `KvmVm` and probably more.

Reply:
I think using `String` here makes a lot of sense. :)